share/spack/gitlab/cloud_pipelines/stacks/ml-linux-x86_64-rocm/spack.yaml

```diff
@@ -0,0 +1,92 @@
+spack:
+  view: false
+  packages:
+    all:
+      require:
+      - target=x86_64_v3
+      - ~cuda
+      - +rocm
+      - amdgpu_target=gfx90a
+    gl:
+      require: "osmesa"
+    py-torch:
+      # Does not yet support Spack-installed ROCm
+      require: ~rocm
```

Comment:
Let's remove this so we can actually test PyTorch too

Reply:
py-torch is still not supported for Spack-installed ROCm

Reply:
In that case, let's remove this and then comment out `py-torch` below so we aren't building yet another CPU copy.

Reply:
We could also add externally installed ROCm to the CI image if that would help with testing.

Reply:
I've commented out py-torch and related packages.